### PR TITLE
Add zkSync to `prices`

### PR DESCRIPTION
**Pull request by lgingerich**

### CODE REVIEW

**Comment by Hosuke** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

GM @lgingerich , thank you for your contribution, but first you may need to follow the instruction and sign the CLA before we continue processing the PR.

**Comment by lgingerich** [comment]
recheck

**Comment by Hosuke** [comment]
BTW currently `zksync` data may not be supported on dune.

**Comment by lgingerich** [comment]
@Hosuke thanks, I see what you mean.

`token-prices-test` fails because CoinPaprika doesn't currently support zkSync Era as a contract platform.

Do you have contact with CoinPaprika so that we can get zkSync added?

**Comment by Hosuke** [comment]
> @Hosuke thanks, I see what you mean.
> 
> `token-prices-test` fails because CoinPaprika doesn't currently support zkSync Era as a contract platform.
> 
> Do you have contact with CoinPaprika so that we can get zkSync added?

I can fix that if the tokens you added are also available on other chains(multi-chains).

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @lgingerich

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -4,10 +4,10 @@
         materialized='table',
         file_format = 'delta',
         tags = ['static', 'dunesql'],
-        post_hook = '{{ expose_spells(\'["ethereum", "solana", "arbitrum", "gnosis", "optimism", "bnb", "avalanche_c", "polygon", "fantom", "celo", "base"]\',
+        post_hook = '{{ expose_spells(\'["ethereum", "solana", "arbitrum", "gnosis", "optimism", "bnb", "avalanche_c", "polygon", "fantom", "celo", "base", "lgingerich"]\',
```
Comment (jeff-dude) [inline models/prices/prices_tokens.sql:7]: looks like copy/paste typo here?

File: models/prices/zksync/prices_zksync_tokens_legacy.sql
Comment (jeff-dude) [inline models/prices/zksync/prices_zksync_tokens_legacy.sql:1]: we no longer need legacy files, as we've turned off legacy spark dbt jobs 🙌 
you can safely remove here (and any future/current PRs)
we will try to get readme updated and some sort of communication out

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/prices/prices_tokens.sql (hunk 1 above)
Comment (lgingerich) [reply models/prices/prices_tokens.sql:7]: ahh yes, thank you! fixed

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**